src/components/NavBar.js

```diff
@@ -0,0 +1,56 @@
+import logoImg from '../images/logo.svg';
+import '../css/NavBar.css';
+import { getProfile } from '../api.js';
+import { useEffect, useState } from 'react';
+
+function NavBar() {
+  const [noneProfile, setNoneProfile] = useState(false);
```

Comment:
boolean 값을 갖는 경우 is~, has~ 로 네이밍하는 경우가 일반적입니다!
```suggestion
  const [hasProfile, setHasProfile] = useState(false);
```